runtime-image/rel8-jdk8-overrides.yaml

```diff
@@ -0,0 +1,42 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap74-openjdk8-runtime-openshift-rhel8"
+description: "The JBoss EAP 7.4 OpenJDK 8 runtime image"
+from: "registry.redhat.io/ubi8/ubi:latest"
+
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-74-openjdk8-runtime-openshift-rhel8-container"
+
+modules:
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "8"
+          - name: os-eap-python
+            version: '3.6'
+          - name: jboss.container.jolokia
+            version: '7'
+          - name: jboss.container.prometheus
+            version: '7'
+
+packages:
+      content_sets_file: content_sets_rhel8.yml
+      install:
+          # required by probes (python 3)
+          - python3-requests
+
+
+osbs:
+  configuration:
+    container:
+      platforms:
+        only:
+          - x86_64
+          #- s390x
+          #- ppc64le
+      compose:
+        pulp_repos: true
+        signing_intent: release
+  repository:
+    name: containers/jboss-eap-74-openjdk8-runtime
```

Comment:
Same comment regarding the repo name, isn't this the existing one? I think these new ones should have a rhel8 in them somewhere.